src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -409,11 +423,13 @@ def _is_poller(obj):
 def create_command(module_name, name, operation,
                    transform_result, table_transformer, client_factory,
                    no_wait_param=None, confirmation=None, exception_handler=None,
-                   formatter_class=None):
+                   formatter_class=None, deprecating_msg=None):
     if not isinstance(operation, string_types):
         raise ValueError("Operation must be a string. Got '{}'".format(operation))
 
     def _execute_command(kwargs):
+        if deprecating_msg:
```

Comment:
Please "promote" the deprecating_info to an attribute on the CliCommand class. This way, it shows up in the command table and can be included/excluded based on usage (tab completion => excluded, feed parser => included, help => it depends)